source.rb

```diff
@@ -0,0 +1,14 @@
+class Source
+  attr_accessor :id, :name, :items
+
+  def initialize(id, name)
+    @id = id
```

Comment:
[Optional] Good job!
If you use the `rand `function to produce id it would be nice because other team members did so.